client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -352,10 +359,14 @@ export default connect(
 				dispatch( fetchReceipt( receiptId ) );
 			},
 			fetchSitePlans( site ) {
-				dispatch( fetchSitePlans( site.ID ) );
+				if ( site ) {
```

Comment:
Is this code needed? [`shouldFetchSitePlans`](https://github.com/Automattic/wp-calypso/blob/1c83fc6086f1b4b94371ba985c0bb5eb1ed88602/client/lib/plans/index.js#L128) already checks for the presence of a site.